pkg/vender/vendor_config_test.go

```diff
@@ -0,0 +1,142 @@
+package vender
```

Comment:
```suggestion
// The package name `vendor` is reserved in Go for dependency management.
// To avoid conflicts, the name `vender` was chosen as an alternative.
package vender
```